Nitrox.Test/Model/Helper/NetHelperTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Net;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NitroxModel.Helper;
+
+namespace Nitrox.Test.Model.Helper
+{
+    [TestClass]
+    public class NetHelperTest
+    {
+        [TestMethod]
+        public void ShouldMatchPrivateIps()
+        {
+            IPAddress.Parse("192.168.2.1").IsPrivate().Should().BeTrue();
```

Comment:
Shouldn't we test all the ips (or just the start and end of the ranges) defined in NetHelper? Why only a few manually selected?

Reply:
Just laziness. Will add the others when I can.